examples/postgres/advanced-blog-cli/src/main.rs

```diff
@@ -1,21 +1,10 @@
 #![deny(warnings)]
 
-extern crate bcrypt;
-extern crate chrono;
 #[macro_use]
 extern crate diesel;
-extern crate dotenv;
-extern crate structopt;
 #[macro_use]
 extern crate structopt_derive;
```

Comment:
I think structopt reexports the derives now and is also usable without the `#[macro_use]` annotation.

Reply:
Updating structopt to 0.3 resolved this.